ran-core/src/main/java/io/ran/Property.java

```diff
@@ -151,7 +151,7 @@ public Annotations getAnnotations() {
 	}
 
 	public static class PropertyList extends ArrayList<Property> {
-		private Map<String, Property> propertyMap = Collections.synchronizedMap(new HashMap<>());
+		private ConcurrentMap<String, Property> propertyMap = new ConcurrentHashMap<>();
```

Comment:
Couldn't the left side stay a Map?

Reply:
[54c5092](https://github.com/persequor-com/ran/pull/30/commits/54c5092f1d6512d675fab78fc5a76a15edd046cc)